### ISSUE TITLE
Using enforcedUserAccount shows login screen on every transaction

### ISSUE THREAD

**Issue by uddinr** [iZettle/sdk-ios#207]
When using enforcedUserAccount on every transaction the user is asked to login again. I would expect if the logged in account is different to the enforcedUserAccount only then should the login window be shown

```
iZettleSDK.shared().enforcedUserAccount = "myemail@test.com"
iZettleSDK.shared().charge(…)
```

**Comment by arikis** [comment]
Hey @uddinr, 

yes, this looks like a bug. We will work on the fix and push it out as new release once we finish. In the meantime you can have a workaround where you lowercase the string before setting it to `enforcedAccount` property.

I will update this issue once we release the fix.

**Comment by uddinr** [comment]
thanks for confirming, yes for now i have lowercased the string

**Comment by arikis** [comment]
Hey @uddinr, this has now been fixed as part of SDK 2.0.2 release!
